frontend/src/views/Landing/index.tsx

```diff
@@ -316,7 +342,7 @@ const LandingPage = (): JSX.Element => {
             </p>
             <div className={styles.heroStatsContainer}>
               <div>
-                <span>unique cells</span>
+                <span>cells</span>
```

Comment:
Also is `cellsHeroNum` now cells instead of unique cells?

CC: @atarashansky I think you updated the cell count most recently?

Thank you!